src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -128,7 +191,7 @@ protected void addOriginEntriesForCorporateCardLines(
      * @param offsetGenerator A BiFunction that may create an offset entry using the generated regular entry and the detailLines List.
      */
     protected void addOriginEntriesForLines(
-            List<OriginEntryFull> originEntries, List<ConcurStandardAccountingExtractDetailLine> detailLines,
+            Consumer<OriginEntryFull> entryConsumer, List<ConcurStandardAccountingExtractDetailLine> detailLines,

```

Comment:
This Consumer (and hence addOriginEntry()) is being invoked in the addOriginEntriesForLines() method.